front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/PipelineController.groovy

```diff
@@ -44,6 +44,18 @@ class PipelineController {
     pipelineDAO.getPipelinesByApplication(application)
   }
 
+  @RequestMapping(value = 'getById/{id:.+}', method = RequestMethod.GET)
```

Comment:
Ok. Reset. A couple questions.
/v2/pipelines would add a new controller. Is that ok? Swagger wants them to have different names (even though they are in different packages). I called it a V2PipelineController. Now the V2 controller has a couple methods and the old controller has the rest. Its fine functionally, but the API is odd, at least when going through the swagger web ui. I dont have stake in that so dont care, but want to verify.

Also, "name" is the pipeline name. Names are scoped to applications. So if you are providing a name, dont you also need the application?